src/handlers/miner_hbbft_handler.erl

```diff
@@ -297,6 +297,40 @@ handle_message(<<BinMsgIn/binary>>, Index, #state{hbbft = HBBFT, skip_votes = Sk
         {ok, {proposed_skip, ProposedRound, IndexRound}} ->
             Skips1 = Skips#{Index => {ProposedRound, IndexRound}},
             {S0#state{skip_votes = Skips1}, []};
+        {ok, {skip, Round, Sigs}} when Round > CurRound ->
```

Comment:
Round > CurRound is correct, right? we can never skip backwards?

Reply:
we can skip backwards, we just can't skip onto the existing round, or we won't make any progress.